.vib/keycloak/cypress/cypress/integration/keycloak_spec.js

```diff
@@ -0,0 +1,87 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows login/logout', () => {
+  cy.login();
+  cy.get('#input-error').should('not.exist');
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('Sign Out').click();
+  cy.contains('#kc-login', 'Sign In');
+});
+
+it('allows changing user data', () => {
+  cy.login();
+  cy.get('.dropdown-toggle').click({ force: true });
+  cy.contains('a', 'Manage account').click({ force: true });
+  cy.contains('p', 'Account console loading').should('not.exist'); //Wait until the loading is finished to continue with actions
+  cy.contains('a', 'Personal').click({ force: true });
+  cy.fixture('users').then((user) => {
+    cy.get('#first-name').clear().type(`${user.newUser.firstName}.${random}`);
+    cy.get('#last-name').clear().type(`${user.newUser.lastName}.${random}`);
+    cy.get('#email-address').clear().type(`${user.newUser.email}`);
+    cy.get('#save-btn').click();
+    cy.contains('.pf-c-alert__title', 'has been updated');
+  });
+});
+
+it('allows creating a new user', () => {
+  cy.login();
+  cy.get('[data-ng-show="access.queryUsers"]').click();
+  cy.get('#createUser').click();
+  cy.fixture('users').then((user) => {
+    cy.get('#username').type(`${user.newUser.username}.${random}`);
+    cy.get('#email').type(`${user.newUser.email}.${random}`);
+    cy.get('#firstName').type(`${user.newUser.firstName}.${random}`);
+    cy.get('#lastName').type(`${user.newUser.lastName}.${random}`);
```

Comment:
You are reusing the same test data here that's already being used in the previous test. This means that the new user and the admin will have the same data (e.g., email), right? I don't think that is very common for an application to allow... Personally, I would add another entry in `users` --> `admin` and `newUser`.